crypto/ake/enclave/src/lib.rs

```diff
@@ -41,16 +41,20 @@ const MAX_AUTH_PENDING_REQUESTS: usize = 64;
 const MAX_PEER_SESSIONS: usize = 64;
 
 /// Maximum number of concurrent sessions to this enclave from router enclaves.
-const MAX_FRONTEND_SESSIONS: usize = 500;
+const MAX_FRONTEND_SESSIONS: usize = 64;
 
 /// Max number of backends that this enclave can connect to as a client.
-const MAX_BACKEND_SESSIONS: usize = 10_000;
+/// Current shard size is 400_000 blocks, with 2 instances per shard.
+/// At 2 million blocks this would require 10 connections. Using 100 connections
+/// allows for 10x increase of 2 million blocks.
+const MAX_BACKEND_SESSIONS: usize = 100;
```

Comment:
This and the `MAX_BACKEND_AUTH_PENDING_REQUESTS` were the heavy hitters.
At 10,000 this used 20,960,000B and `MAX_BACKEND_AUTH_PENDING_REQUESTS` used 12,480,000B. 
These values represent how many shard replicas a router needs to talk to.